src/sortStudents.ts

```diff
@@ -1,16 +1,59 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+export type SortOrder = 'asc'|'desc';
+
+function getAverageGrade(grades:number[]):number {
+  const sumOfGrades = grades.reduce((sum, grade) => sum + grade, 0);
+  const gradesCount = grades.length;
+
+  return sumOfGrades / gradesCount;
 }
 
-// create SortOrder type
-export type SortOrder;
+export function sortStudents(
+  students:Student[],
+  sortBy:SortType,
+  order:SortOrder,
+):Student[] {
+  const sortedStudents = [...students];
+  const currentOrder = order === 'asc';
+
+  sortedStudents.sort((a, b) => {
```

Comment:
you can return immediately
```suggestion
  return sortedStudents.sort((a, b) => {
```